lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -374,3 +413,63 @@ def _discard_tunnel_flows(self, precedence:int, i_teid:int,
             flows.delete_flow(self._datapath, self.tbl_num, match,
                               priority=priority + 1)
 
+    @set_ev_cls(ofp_event.EventOFPPacketIn, MAIN_DISPATCHER)
+    def _packet_in_handler(self, ev):
+        msg = ev.msg
+        pkt = packet.Packet(msg.data)
+        pkt_ipv4 = pkt.get_protocol(ipv4.ipv4)
```

Comment:
Lets add some rudimentary checks here to make sure pkt_ipv4 is not None, in case we get pkt in for some random packet somehow

Reply:
Changes done.